FirebaseSessions/Tests/Unit/SessionCoordinatorTests.swift

```diff
@@ -84,4 +90,43 @@ class SessionCoordinatorTests: XCTestCase {
     XCTAssertEqual(fireLogger.loggedEvent, event)
     XCTAssertFalse(resultSuccess)
   }
+
+  func test_eventNotDropped_handlesAllEventsAllowed() throws {
+    identifiers.mockAllValidIDs()
+
+    let event = SessionStartEvent(identifiers: identifiers, appInfo: appInfo, time: time)
+
+    sampler.sessionSamplingRate = 1.0
+    var resultSuccess = true
+    coordinator.attemptLoggingSessionStart(event: event) { result in
+      switch result {
+      case .success(()):
+        resultSuccess = true
+      case .failure:
+        resultSuccess = false
+      }
+    }
+
+    XCTAssertTrue(resultSuccess)
+  }
+
+  func test_eventDropped_EventsSampled() throws {
```

Comment:
Similar to earlier comment, `sample` is a confusing word, and to follow the same pattern as the previous test, perhaps `test_eventDropped_handlesZeroSamplingRate`?

Reply:
Good point. Updated it.